tests/integration/test_alert_rules.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
```

Comment:
Consider renaming file e.g. `test_multiple_rule_providing_apps`

Reply:
Will do it in: https://github.com/canonical/loki-k8s-operator/issues/103